examples/using-JSON-YAML-source/src/components/header.js

```diff
@@ -0,0 +1,42 @@
+import { Link } from 'gatsby'
+import PropTypes from 'prop-types'
+import React from 'react'
+
+const Header = ({ siteTitle }) => (
+  <div
```

Comment:
Change this wrapper DIV to a `header` element (it's a change that I also plan to make upstream in the starter):
```
<header
    style={{
      background: `rebeccapurple`,
      marginBottom: `1.45rem`,
    }}
>
```